examples/gallery/embellishments/inset.py

```diff
@@ -14,12 +14,17 @@
 # "brown", the water to "lightblue", the shorelines width to "thin", and adding a frame
 fig.coast(region="MG+r2", land="brown", water="lightblue", shorelines="thin", frame="a")
 # Create an inset, setting the position to top left, the width to 3.5 centimeters, and
-# the x- and y-offsets to 0.2 centimeters. The margin is set to 0, and the border is "gold".
-with fig.inset(position="jTL+w3.5c+o0.2c", margin=0, box="+pgold"):
+# the x- and y-offsets to 0.2 centimeters. The margin is set to 0, and the border is "gold"
+# with a pen size of 1.5p.
+with fig.inset(position="jTL+w3.5c+o0.2c", margin=0, box="+p1.5p,gold"):
     # Create a figure in the inset using coast. This example uses the azimuthal
     # orthogonal projection centered at 47E, 20S. The land color is set to "gray" and
-    # Madagascar is highlighted in "red".
+    # Madagascar is highlighted in "red3".
     fig.coast(
-        region="g", projection="G47/-20/3.5c", land="gray", water="white", dcw="MG+gred"
+        region="g",
+        projection="G47/-20/3.5c",
```

Comment:
We should use `?` to let GMT automatically determine the map width in the inset.
```suggestion
        projection="G47/-20/?",
```